corehq/apps/export/models/new.py

```diff
@@ -1868,9 +1868,18 @@ def _generate_schema_from_xform(cls, xform, langs, app_id, app_version):
         schema = cls()
         question_keyfn = lambda q: q['repeat']
 
-        question_groups = [(x, list(y)) for x, y in groupby(
-            sorted(questions, key=question_keyfn), question_keyfn
```

Comment:
Another option would be to use a key function with `sorted` that always returns a string. Something like `question_keyfn_str = lambda q: q['repeat'] or ''.

It would need some more tweaking to avoid unexpected groupings if it's expected that values of `q['repeat']` might be either `None` or `''`.